mysql-test/suite/innodb/t/percona_create_table_8k.test

```diff
@@ -0,0 +1,88 @@
+--echo #
```

Comment:
Please mark this MTR test case as `big-test`

Reply:
Actually it might make sense to *also* mark it as requiring standalone execution - see not_parallel.inc, so it is not run concurrently with other big tests (I assume it requires quite some resources?)

Reply:
Do we run not_parallel tests on jenkins?

Reply:
@dlenev I verified that it indeed scheduled by one last worker in mtr. So we need not special run initiated by Jenkins.